### ISSUE TITLE
Reconsider heading usage

### ISSUE THREAD

**Issue by swissspidy** [swissspidy/oEmbed-API#57]
From @afercia [on WordPress Slack](https://wordpress.slack.com/archives/accessibility/p1441663361000385):

> About the oEmbed API feature plugin, below it's a screenshot with the headings structure in a post which embeds a WP post from an external site (Rian's test server). The embedded content is inside an iframe which uses H1 and H2 headings which may disrupt the heading structure in a page. Of course, screen readers get both H1 headings when pressing "1". Other oEmbed providers use iframes but looks like they don't use headings at all (facebook, twitter). Thoughts? We should agree on a recommendation. Headings or not headings?

![xqwzlr-nn3](https://cloud.githubusercontent.com/assets/841956/9738453/dfb2c2ae-564b-11e5-8e2f-9e6ad7599386.png)

I only found [an old thread](http://webaim.org/discussion/mail_thread?thread=4330) which suggests using no headings at all, so that's what I'm leaning more towards.

**Comment by joedolson** [comment]
Using a lower heading level (H2 or H3) could be correct in the hierarchy of some sites, but an H1 will never be right in any heading hierarchy. I'd also be in favor of going with no heading for this; semantically, while the title is a heading when in context of it's own page, I'm not sure that's also true when it's been embedded elsewhere. In that context, I feel like it behaves like just a block of content, that should simply fit into the hierarchy of the page where it's being used.
